### PR TITLE
Style edit topic page

### PR DESCRIPTION
**Pull request by local-girl**
Again, this form is the same as as create topic

### CODE REVIEW

**Comment by fisx** [comment]
LGTM.  I shuffled the code around a bit.  Ready to merge IMO.

File: src/Frontend/Page/Topic.hs [hunk 1]
```diff
@@ -162,21 +162,26 @@ instance FormPage CreateTopic where
...
+createOrEditTopic :: Monad m => View (HtmlT m ()) -> [Idea] -> HtmlT m ()
+createOrEditTopic v ideas = do
+    label_ $ do
+        span_ [class_ "label-text"] "Wie soll der Titel des Themas lauten?"
+        inputText_ [class_ "m-small", placeholder_ "z.B. Computerraum"]
+            "title" v
+    label_ $ do
+        span_ [class_ "label-text"] "Beschreiben Sie das Thema"
+    -- FIXME I want a placeholder here too
+        DF.inputTextArea Nothing Nothing "desc" v
```
Comment (andorp) [inline src/Frontend/Page/Topic.hs]: On Edit topic the exiting text of the topic should be rendered as the content in the  `inputTextArea`.

Comment (fisx) [inline src/Frontend/Page/Topic.hs]: > On Edit topic the exiting text of the topic should be rendered as the content in the  `inputTextArea`.

yes, but in the 'create' mode, it doesn't.  and we don't want to have it as the default value either.  there is a 'placeholder_' attribute, but it's not supported by DF.inputTextArea, so we need to write another wrapper.

Comment (andorp) [inline src/Frontend/Page/Topic.hs]: @fisx show me offline, that works as expected. Sorry for the noise.

**Pull request merged**